nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/json/JsonTreeReader.java

```diff
@@ -39,11 +39,11 @@
 
 @Tags({"json", "tree", "record", "reader", "parser"})
 @CapabilityDescription("Parses JSON into individual Record objects. The Record that is produced will contain all top-level "
-    + "elements of the corresponding JSON Object. "
-    + "The root JSON element can be either a single element or an array of JSON elements, and each "
-    + "element in that array will be treated as a separate record. "
-    + "If the schema that is configured contains a field that is not present in the JSON, a null value will be used. If the JSON contains "
-    + "a field that is not present in the schema, that field will be skipped. "
+        + "elements of the corresponding JSON Objects. The reader does not require the flow file content to be well-formed JSON; rather "
```

Comment:
I would make the same comment here as above, regarding the phrasing that the flow file content need not be well-formed JSON.